subprojects/core/core.gradle.kts

```diff
@@ -152,7 +152,6 @@ dependencies {
 
 strictCompile {
     ignoreRawTypes() // raw types used in public API
```

Comment:
Can this be removed, too?

Reply:
We can do that later. We have it in a few projects at the moment which have a lot of RAW API. We could add a SuppressWarning annotation to all of these, but I am not sure as there are so many. I would rather see that we address https://github.com/gradle/gradle-private/issues/3073 first I think.